src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -513,83 +514,78 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 	async openEditor(editor: IEditorInput | IUntypedEditorInput, optionsOrGroup?: IEditorOptions | OpenInEditorGroup, group?: OpenInEditorGroup): Promise<IEditorPane | undefined> {
 		const result = this.doResolveEditorOpenRequest(editor, optionsOrGroup, group);
 		if (result) {
-			const [resolvedGroup, resolvedEditor, resolvedOptions] = result;
+			const [resolvedGroup, resolvedOptions] = result;
 
 			// Override handling: request override from override service
 			if (resolvedOptions?.override !== EditorOverride.DISABLED) {
-				const overrideResult = await this.editorOverrideService.resolveEditorOverride(resolvedEditor, resolvedOptions, resolvedGroup);
+				const overrideResult = await this.editorOverrideService.resolveEditorOverride(editor, resolvedGroup);
```

Comment:
This is weird. `resolveEditorOverride` only seems to accept `IUntypedEditorInput` but you are passing in `EditorInput`.